pyproject.toml

```diff
@@ -67,23 +67,23 @@ python = ">=3.6,<3.9"
 coloredlogs = ">=10,<15"
 sanic = ">=19.12.2,<21.0.0"
 sanic-cors = "^0.10.0"
-requests = "^2.23"
+requests = ">=2.23.0,<2.26.0"
 
 [tool.poetry.dev-dependencies]
 pytest-cov = "^2.10.1"
-coveralls = "^2.1.2"
-pytest = "^5.4.1"
+coveralls = "^2.2.0"
+pytest = "^6.1.2"
 black = "^20.8b1"
 flake8 = "^3.8.2"
 flake8-docstrings = "^1.5.0"
 pytest-sanic = "^1.6.1"
-questionary = "^1.8.1"
+questionary = ">=1.7.0,<1.9.0"
 towncrier = "^19.2.0"
-toml = "^0.10.0"
+toml = "^0.10.2"
 pep440-version-utils = "^0.3.0"
 semantic_version = "^2.8.5"
 typing-extensions = "^3.7.4"
-mypy = "^0.782"
+mypy = "^0.790"
 
 # uvloop >= 0.15.0 is not compatible with Python 3.6 and not available on Windows
```

Comment:
is `poetry>1.10` smart enough to detect the incompatibility of `uvloop==0.15.0` with Python 3.6 itself?

Reply:
it blows up if only `markers = "sys_platform != 'win32'"` is there...